src/modules/rewards/components/RewardActivities.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC } from 'react'
+import { css } from 'styled-components';
+import { Box, Button, Text } from 'blocks';
+
+export type RewardActivitiesProps = {
+  onGetStarted: () => void;
+}
+
+const RewardActivities: FC<RewardActivitiesProps> = ({ onGetStarted }) => {
+
+  const rewardsArray = [
+    {
+      title: 'Follow @PushProtocol on X',
+      points: 10000,
+      buttonText: 'Follow'
+    },
+    {
+      title: 'Join Push Discord',
+      points: 10000,
+      buttonText: 'Join Discord'
+    },
+    {
+      title: 'Send 20 Messages using Push Chat',
+      points: 1500,
+      buttonText: 'Claim',
+      disabled: true
+    },
+    {
+      title: 'Subscribe to 3 Channels on PushX',
+      subtitle: 'Visit app.push.org/channels and opt-in to any 3 channels.',
+      points: 1500,
+      buttonText: 'Follow'
+    }
+  ]
+  
+  return (
+    <Box 
+      display='flex'
+      flexDirection='column'
+      gap='s1'
+    >
+      <Text variant='h4-bold'>Activities</Text>
+
+      {rewardsArray.map((item, i) => (
+        <Box
```

Comment:
This Box component needs to go to RewardsActivitiesListItem.tsx

```suggestion
        {rewardsArray.map((item, i) => (
        <RewardsActivitiesListItem key={key}
```